src/store/concepts/account/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers } from 'redux'
+
+import { ACCOUNT_SET_USERNAME } from './types'
+
+const username = (state = null, action) => {
+  switch (action.type) {
+    case ACCOUNT_SET_USERNAME:
```

Comment:
`SET_USERNAME` because you already have `const namespace = 'account'`

Reply:
Fixed